CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * [#6029](https://github.com/bbatsov/rubocop/issues/6029): Fix a false positive for `Lint/ShadowedArgument` when reassigning to splat variable. ([@koic][])
 * [#6036](https://github.com/rubocop-hq/rubocop/issues/6036): Make `Rails/BulkChangeTable` aware of string table name. ([@wata727][])
 * [#5467](https://github.com/bbatsov/rubocop/issues/5467): Fix a false negative for `Style/MultipleComparison` when multiple comparison is not part of a conditional. ([@koic][])
+* Fix `Lint/RedundantWithObject` error on missing parameter to `each_with_object`. ([@Vasfed][])
```

Comment:
Since there is no related issue, adding a link to this PR at the beginning is kind to users who sees CHANGELOG.md.

```diff
-* Fix `Lint/RedundantWithObject` error on missing parameter to `each_with_object`. ([@Vasfed][])
+* [#6042](https://github.com/bbatsov/rubocop/pull/6042): Fix `Lint/RedundantWithObject` error on missing parameter to `each_with_object`. ([@Vasfed][])
```

So, other changes looks good to me.

Reply:
@koic Ok :)
Should commits be squashed after PR is already reviewed? Guide only mentions prior process, but for minor documentation changes it feels strange to run about an hour of tests